src/pages/Homepage/DefaultHomepage.vue

```diff
@@ -1,6 +1,5 @@
 <template>
 	<div>
-		<top-message-contentful />
```

Comment:
I don't believe this should be here. This component doesnt even render without content prop